<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_room">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>房间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_roomid">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_t_aircondi">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_t">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>空调温度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_On_Off">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开机</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fee">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>费用</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_fee">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>显示面板</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>控制面板</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_mode_control">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>212</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_speed_control">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>风速</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_speed_show">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>风速</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mode_show">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_mode">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_speed">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_t_control">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>温度修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_checkout">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退房</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_mode_cold">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>制冷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_mode_hot">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>制热</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_speed_low">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>低速</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_speed_middle">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>中速</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_speed_high">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>高速</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_t_room">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_t_room">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>房间温度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_change_aircondi">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_changeT">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
